timemachine

```diff
@@ -181,28 +181,31 @@ RSYNC_PARTIAL=".partial"
 # [resume]      --partial-dir:        Where to store unfinished files for resume
 # [atomic]      ${BACKUP_INPROGRESS}: Tmp dest dir for atomic operations
 
-if test -L "${DEST}/${BACKUP_LATEST}"; then
-	BTYPE="incremental"
-else
-	BTYPE="full"
-fi
-
-logmsg "Starting ${BTYPE} backup"
-logmsg "\$ rsync $* ${SRC} ${DEST}/${BACKUP_INPROGRESS}"
+BTYPE=
 
 # Only link destination if it already exists
 if test -L "${DEST}/${BACKUP_LATEST}"; then
+	BTYPE="incremental"
```

Comment:
This change might duplicate a bit of code, but definitely ensures the log message output is what actually happens